LoopKit/Extensions/HKQuery.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  HKQuery.swift
+//  LoopKit
+//
+//  Created by Rick Pasetto on 6/22/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import HealthKit
+
+public var excludeHealthKitDataFromOtherApps = false
+
+extension HKQuery {
+    public class func predicateForSamplesForLoop(withStart startDate: Date?, end endDate: Date?, options: HKQueryOptions = []) -> NSPredicate {
```

Comment:
`ForLoop` -> `ForCurrentApp`

Reply:
@ps2 Yeah, funny you should suggest that.  That's what I _first_ had it at, but because it is not only for the "current app" for DIY loop, I felt it was a misnomer and misleading.  I'm ok with changing it back, if you think it is fine.

Reply:
Oh, yeah, it's checking the the bool var inside. In that case, you don't need to rename at all.

Reply:
I mean the original name `predicateForSamples` still works, you don't need to append anything. I think referring to the "Loop" app in LoopKit is almost never correct.